### PR TITLE
Add docs for target-info

### PR DESCRIPTION
**Pull request by JSCU-CNI**
This PR adds documentation for `target-info` and adds a `watch` command to the Makefile.

### CODE REVIEW

File: docs/source/tools/target-info.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+``target-info`` enables you to quickly view basic system information of your targets.
+
+The following output types are supported:
+
+- Human readable format (default)
+- Record output (``-r``)
+- JSON output (``-j`` and ``-J``)
+
+The command below demonstrates ``target-info`` output on an :doc:`/tools/acquire` image.
```
Comment (Schamper) [inline docs/source/tools/target-info.rst:12]:
```suggestion
The command below demonstrates ``target-info`` output on an :doc:`/tools/acquire` file.
```

File: docs/source/tools/target-info.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    Ips            : 10.13.37.1
+    Os family      : windows
+    Os version     : Windows Server 2016 Datacenter (NT 10.0) 12345.6789
+    Architecture   : amd64-win64
+    Language       : en_US
+    Timezone       : Europe/Berlin
+    Install date   : 2020-04-29 10:48:49+00:00
+    Last activity  : 2023-01-02 13:37:00+00:00
+
+The command below demonstrates ``target-info`` output in JSON format using the option ``-j``. Use the option ``-J`` to generate compressed JSON output.
```
Comment (Schamper) [inline docs/source/tools/target-info.rst:31]:
```suggestion
The command below demonstrates ``target-info`` output in JSON format using the option ``-j``. Use the option ``-J`` to generate JSON line output.
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: docs/source/tools/target-info.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+The command below demonstrates ``target-info`` output using the option ``-r`` in :ref:`record <overview/index:records>` format.
+
+.. code-block:: console
+    :class: pre-wrap
+
+    $ target-info -r target.vmx
+
+    <target/info hostname='DESKTOP01' domain='DOMAIN' last_activity=2023-10-16 14:06:33.678421+00:00 install_date=2023-02-01 15:01:07+00:00 ips=[net.ipaddress('10.13.37.3')] os_family='windows' os_version='Windows 10 Pro (NT 10.0) 19045.2006' architecture='amd64-win64' language=['en_GB', 'en_US'] timezone='Europe/Berlin' disks=["{'type': 'VmdkContainer', 'size': 34359738368}"] volumes=["{'name': 'EFI system partition', 'size': 209714688, 'fs': 'FatFilesystem'}", "{'name': 'Microsoft reserved partition', 'size': 134217216, 'fs': 'NoneType'}", "{'name': 'Basic data partition', 'size': 34013707776, 'fs': 'NtfsFilesystem'}"] children=["{'type': 'wsl', 'path': 'C:\\\\Users\\\\user\\\\AppData\\\\Local\\\\Packages\\\\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\\\\LocalState\\\\ext4.vhdx'}"]>
```
Comment (Schamper) [inline docs/source/tools/target-info.rst:68]:
```suggestion
The command below demonstrates ``target-info`` output using the option ``-r`` in :ref:`record <overview/index:records>` format.

.. code-block:: console
    :class: pre-wrap

    $ target-info -r target.vmx

    <target/info hostname='DESKTOP01' domain='DOMAIN' last_activity=2023-10-16 14:06:33.678421+00:00 install_date=2023-02-01 15:01:07+00:00 ips=[net.ipaddress('10.13.37.3')] os_family='windows' os_version='Windows 10 Pro (NT 10.0) 19045.2006' architecture='amd64-win64' language=['en_GB', 'en_US'] timezone='Europe/Berlin' disks=["{'type': 'VmdkContainer', 'size': 34359738368}"] volumes=["{'name': 'EFI system partition', 'size': 209714688, 'fs': 'FatFilesystem'}", "{'name': 'Microsoft reserved partition', 'size': 134217216, 'fs': 'NoneType'}", "{'name': 'Basic data partition', 'size': 34013707776, 'fs': 'NtfsFilesystem'}"] children=["{'type': 'wsl', 'path': 'C:\\\\Users\\\\user\\\\AppData\\\\Local\\\\Packages\\\\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\\\\LocalState\\\\ext4.vhdx'}"]>

The command below demonstrates ``target-info`` output in JSON format using the option ``-j``. Use the option ``-J`` to generate JSON line output.

.. code-block:: console

    $ target-info -j target.vmx
.. code-block:: json

    {
        "disks": [
            {"type": "VmdkContainer", "size": 21474836480}
        ],
        "volumes": [
            {"name": "part_00100000", "size": 1048064, "fs": "NoneType"},
            {"name": "part_00200000", "size": 1902116352, "fs": "ExtFilesystem"},
            {"name": "part_71800000", "size": 19569573376, "fs": "NoneType"},
            {"name": "ubuntu--vg-ubuntu--lv", "size": 10737418240, "fs": "ExtFilesystem"}
        ],
        "children": [],
        "hostname": "SERVER01",
        "domain": "domain.local",
        "ips": ["10.13.37.2"],
        "os_family": "linux",
        "os_version": "Ubuntu 22.04.1 LTS",
        "architecture": "x86_64-linux",
        "language": ["en_US"],
        "timezone": "Etc/UTC",
        "install_date": "2023-02-02 12:07:32.646006+00:00",
        "last_activity": "2023-10-16 15:06:34.472182+00:00"
    }
```
Best to use the same order as you list the options in at the beginning.

**Review by Schamper — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -23,6 +25,13 @@ If you run into build issues, clean the Sphinx output directory and AutoAPI outp
 cd docs && make clean
 ```
 
+When writing documentation you can use the convenience command below. This rebuilds
```
Comment (Schamper) [inline README.md:28]: @Miauwkeru I know you played around a fair bit with the build stuff for Sphinx, do you have an opinion on these additions?

Comment (JSCU-CNI) [reply README.md:28]: When used correctly `sphinx-autobuild` should not "keep writing stuff". I encountered this too, you need to have built the project manually at least once (using `make html`) before using the `make watch` command. 

The behavior afterwards should be that only files you change will be re-built by sphinx.

Comment (JSCU-CNI) [reply README.md:28]: Added in [3417461](https://github.com/fox-it/dissect-docs/pull/36/commits/3417461df1dfa2e39d04fee6494d0484ab5fe177)

**Review by Schamper — APPROVED** [review]

**Pull request merged**